index.html

```diff
@@ -1,67 +1,271 @@
-            /></a>
-          </li>
-        </ul>
-      </nav>
-    </header>
 
-    <section class="my-self">
-      <div class="myself-header">
-        <h1 class="my-name">Hey There. I’m OyelakinG</h1>
-        <h4 class="my-work">I’am a Software Developer</h4>
-        <p class="my-text">
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="css/my_portfolio.css" />
+</head>
+
+<body>
+  <header id="header" class="header">
+    <div class="logo"><a href="#" class="logo-link">John Doe</a></div>
+    <nav id="navbar" class="navbar">
+      <div class="burger">
+        <img src="assets/images/burger.png" alt="" class="d-none-lg" />
+      </div>
+      <ul class="d-none nav-list d-flex-lg list-style-none">
+        <li class="nav-item">
+          <a href="#" class="nav-link">Portfolio</a>
+        </li>
+        <li class="nav-item">
+          <a href=".about-us" class="nav-link">About</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Contact</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link"><img src="assets/images/message-box.png" alt="" /></a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="my-self">
+    <div class="myself-header">
+      <h1 class="my-name">Hey There. I’m OyelakinG</h1>
+      <h4 class="my-work">I’am a Software Developer</h4>
+      <p class="my-text">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <div class="media-icons">
+      <ul class="list-style-none d-flex d-block-lg">
+        <li class="github"><img src="assets/images/github.png" alt="" /></li>
+        <li class="linkedin margin-left-5 margin-top-lg">
+          <img src="assets/images/linked_in.png" alt="" />
+        </li>
+        <li class="sure margin-left-5 margin-top-lg">
+          <img src="assets/images/sure.png" alt="" />
+        </li>
+        <li class="twitter margin-left-5 margin-top-lg">
+          <img src="assets/images/twitter.png" alt="" />
+        </li>
+        <li class="mozilla margin-left-5 margin-top-lg">
+          <img src="assets/images/Vector.png" alt="" />
+        </li>
+      </ul>
+    </div>
+    <div class="group1 pos-absolute d-none-lg"></div>
+    <!-- <div class="group2 pos-absolute d-none d-block-lg"></div> -->
+  </section>
+
+  <!-- work section -->
+  <section id="work-section" class="work-section">
```

Comment:
- [ ] As indicated by the previous reviewer, the work section is supposed to have a `display: grid` and looking at your CSS in `lines 30 to 32`, you defined a `d-grid` class there, which is good CSS practice. I think you need to add this class in your index.html line 68.